src/components/Notifications/NotificationList/NotificationList.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import NotificationListItem from "./NotificationListItem/NotificationListItem";
+
+const List = styled.ul`
+  width: 80%;
+  list-style: none;
+  padding: 0;
+  margin: 0 auto;
+  font-size: 0.75rem;
+`;
+
+function NotificationList({ notifications }) {
+  return (
+    <List>
+      {notifications.map(({ user, message, date_created }) => {
```

Comment:
Is there a reason to use underscores for `date_created` instead of camel casing? Since we are using camel casing in other places. If there is a reason then it's fine.